### PR TITLE
Blog articles for 25th September 2017

### PR DESCRIPTION
**Pull request by archpndy**
Changed the heading tags.

### CODE REVIEW

File: blog_src/_data/authors.yml [hunk 1]
```diff
@@ -263,3 +263,9 @@ vibhoregit:
   author_image: vibhore.jpg
   user_description: '"Everyone is special" - a classic example of Oxymoron...'
   email: jainvibhore1997@gmail.com
+archpndy:
+  name: Archit Pandey
+  github: archpndy
+  author_image: archit.jpg
+  user_description: ''
```
Comment (salman-bhai) [inline blog_src/_data/authors.yml]: Add user description. Don't leave it as blank please!

**Comment by salman-bhai** [comment]
Also change the PR description when you add his article as well!

File: blog_src/_posts/2017-09-25-quantum-computers.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+ 
+The first ever made quantum computer (fig 1)
+
+## Breaking The Myth of Superposition Theorem
+
+Before dwelling into the topic, I would like to make one thing clear the superposition as taught by many is not a particle being at different position at a given time but it is the probability of the particle wave density being distributed at different positions in space time but does exists at only one found position at that given time.
+
+### Qubits
+
+A quantum computer maintains a sequence of qubits. A single qubit can represent a one, a zero, or any quantum superposition of those two qubit states a pair of qubits can be in any quantum superposition of 4 states, and three qubits in any superposition of 8 states. In general, a quantum computer with **N* qubits can be in an arbitrary superposition of up to 2^N{\displaystyle 2^{n}}different states simultaneously.
```
Comment (salman-bhai) [inline blog_src/_posts/2017-09-25-quantum-computers.md]: I think you missed a * here which is why the rest of the article is coming as bold! Fix it here!

File: blog_src/_posts/2017-09-25-quantum-computers.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+layout: post
+title: "Quantum Computers and The Future of Computation"
+author_github: sampathkoti
+date: 2017-09-18 10:00:30
+image: '/assets/img/'
+description: 'A look at the future of computing'
+tags:
+- Quantum Computing
+categories:
+- CAS/SPS
```
Comment (salman-bhai) [inline blog_src/_posts/2017-09-25-quantum-computers.md]: Change categories to Diode! Check other articles to see what it exactly is!

File: blog_src/_posts/2017-09-25-quantum-computers.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+Quantum Mechanics a realm of physics where the tiny subatomic particles reside and whose physics is beyond the perception of the classical physical world.
+
+### 1] What makes this realm so weird and different in working?
+
+The behaviour working of these particles   was resolved by proposing of a mathematical model governed by the principle of Superposition, entanglement and tunnelling.
+
+Quantum Computers makes use of these physical phenomena to do it calculation and this is where the speciality of the futuristic computing machines lies.
+ 
+The first ever made quantum computer (fig 1)
```
Comment (salman-bhai) [inline blog_src/_posts/2017-09-25-quantum-computers.md]: Where is fig1 ?

**Comment by archpndy** [comment]
How do I change the description?

File: blog_src/_posts/2017-09-25-quantum-computers.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+There is such a thing as quantum cryptography, which uses photon-based qubits to securely transmit information encoded into the quantum states of particles. This quantum communication makes it possible for the recipient to detect attempts to intercept incoming messages. And it isn't exactly new.
+
+Its applications include what's called the quantum key distribution(QKD). Basically, it uses quantum communication to share keys securely, which will be used to decrypt messages sent over conventional networks. Unfortunately, low bandwidth makes the system currently untenable, despite having been demonstrated to work in several cities.
+
+This is just one possible work around. Other methods are being developed, including code-based cryptography and lattice-based cryptography. In any case, there's time to improve it. In the same way that quantum computing is still being refined, network infrastructure can be improved to allow for quantum secure cryptography.
+
+
+## For The Interested Ones
+
+1. Shor's algorithm: [**www.youtube.com/watch?v=FA21Dj2l3Ac**](http://www.youtube.com/watch?v=FA21Dj2l3Ac)
```
Comment (salman-bhai) [inline blog_src/_posts/2017-09-25-quantum-computers.md]: Change this to `[Shor's algorithm](http://www.youtube.com/watch?v=FA21Dj2l3Ac)`

File: blog_src/_posts/2017-09-25-quantum-computers.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+2. Maze analogy: [**https://www.nature.com/articles/ncomms11682**](https://www.nature.com/articles/ncomms11682)
```
Comment (salman-bhai) [inline blog_src/_posts/2017-09-25-quantum-computers.md]: Change this to `: [Maze analogy](https://www.nature.com/articles/ncomms11682)`

File: blog_src/_posts/2017-09-25-quantum-computers.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+
+3. Quantum crypto: [**http://www.popsci.com/what-is-quantum-cryptography**](http://www.popsci.com/what-is-quantum-cryptography)
```
Comment (salman-bhai) [inline blog_src/_posts/2017-09-25-quantum-computers.md]: Change this to `[Quantum crypto](http://www.popsci.com/what-is-quantum-cryptography)`

File: blog_src/_posts/2017-09-25-quantum-computers.md [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+
+4. Cryptography: [**https://www.youtube.com/watch?v=Kf9KjCKmDcU&amp;list=PL7AEDF86AABA1AA9A**](https://www.youtube.com/watch?v=Kf9KjCKmDcU&amp;list=PL7AEDF86AABA1AA9A)
```
Comment (salman-bhai) [inline blog_src/_posts/2017-09-25-quantum-computers.md]: Change this to `[Cryptography](https://www.youtube.com/watch?v=Kf9KjCKmDcU&amp;list=PL7AEDF86AABA1AA9A)`

File: blog_src/_posts/2017-09-25-quantum-computers.md [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+
+5.  IBM: [**https://www.youtube.com/watch?v=jf7D8snlsnQ&amp;feature=youtu.be**](https://www.youtube.com/watch?v=jf7D8snlsnQ&amp;feature=youtu.be)
```
Comment (salman-bhai) [inline blog_src/_posts/2017-09-25-quantum-computers.md]: Change this to ` [IBM](https://www.youtube.com/watch?v=jf7D8snlsnQ&amp;feature=youtu.be)`

**Pull request merged**